conjur/login/credential_data.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialData module
+
+This module represents an object that holds netrc data
+"""
+
+# pylint: disable=too-few-public-methods
+class CredentialData:
+    """
+    Used for setting user input data to login to Conjur
+    """
+
+    def __init__(self, machine=None, login=None, password=None):
+        self.machine = machine
+        self.login = login
+
+        # Purposefully not named the same for clarity in code. .netrc file format standard
+        # uses password but this value is actually the user/host api key. This convention should
+        # be kept. See https://www.labkey.org/Documentation/wiki-page.view?name=netrc especially
+        # the 'Use API Keys' section
+        self.password = password
+
+    # pylint: disable=line-too-long
+    def __repr__(self):
+        return "{'machine': %r, 'login': %r, 'password': %r}" % (self.machine,
+                                                                 self.login,
+                                                                 self.password)
```

Comment:
You need to obfuscate the password when you print it.

Reply:
like print only first 2 letters and replace the rest with ellipsis:
```suggestion
                                                                 self.password[:2] + "...")
```
So if the password is `P@ssw0rd`, you will see: `P@...` in the print.

Reply:
Great catch!!! Is this enough in terms of security? I can just write `***` because the actual value doesnt really need to be printed

Reply:
You can consult with @moticless if you want, how to correctly handle password.
but for sure, you shouldn't print it to anywhere, unless it is safe.